src/test/java/sorinfoca/Tests/Tema6/DemoQaWindow.java

```diff
@@ -0,0 +1,99 @@
+package sorinfoca.Tests.Tema6;
+
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.interactions.Actions;
+import sorinfoca.driver.BrowserManager;
+import sorinfoca.utils.FileUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Set;
+
+public class DemoQaWindow {
+    static ChromeDriver driver = null;
+    private static File screenshot;
+
+    public static void main(String[] args) {
+        navigateToDemoQaWindowPage();
+        try {
+            clickNewTabButton();
+            interactWithElementsOnParentTab();
+        } catch (NoAlertPresentException e) {
+            FileUtils.takeScreenshot(driver, "alert");
```

Comment:
Instead of "alert" use "window" or something more meaningful to the class you are getting the exception from. Thanks!